app/src/main/java/org/simple/clinic/summary/PatientSummaryUpdate.kt

```diff
@@ -101,10 +101,13 @@ class PatientSummaryUpdate : Update<PatientSummaryModel, PatientSummaryEvent, Pa
 
   private fun contactDoctorClicked(model: PatientSummaryModel): Next<PatientSummaryModel, PatientSummaryEffect> {
     return when (model.teleconsultInfo) {
+      // TODO (SM): Check phone numbers before dispatching
+      //  LoadPatientTeleconsultationInfo to show contact doctor sheet
```

Comment:
I am assuming this means we will first show the sheet to pick a number (if more than one number is present) and then trigger the event to load the teleconsult information once the number is selected, right?

Reply:
Yeah, currently we load the teleconsultation info once the contact button is clicked. We will be following a similar pattern.